example/lib/example_polls.dart

```diff
@@ -40,7 +40,11 @@ class _ExamplePollsState extends State<ExamplePolls> {
                 pollId: poll['id'].toString(),
                 // hasVoted: hasVoted.value,
                 // userVotedOptionId: userVotedOptionId.value,
-                onVoted: (PollOption pollOption, int newTotalVotes) {
+                onVoted: (PollOption pollOption, int newTotalVotes) async {
+                  await Future.delayed(const Duration(seconds: 2));
```

Comment:
Could you explain why this is necessary?

Reply:
Just delaying to mimic a Server delay in response. That's were the loadingWidget comes into play...
Basically 
I supposed that the [onVoted] takes an async callback that returns boolean as success status. It should contain code to perform an http request to a server eg. 'api/post/1/poll/3'. 
If the request is successful,  the call back returns true. And the poll widget rebuilds, until then a loading widget is placed.
If the request is unsuccessful, return false. The poll widget doesn't change.